reflex/utils/format.py

```diff
@@ -628,3 +628,45 @@ def unescape_double_quotes_in_var(m: re.Match) -> str:
         string=value,
         flags=re.VERBOSE,
     )
+
+
+def format_data_editor_column(col: str | dict):
+    """Format a given column into the proper format.
+
+    Args:
+        col: The column.
+
+    Raises:
+        ValueError: invalid type provided for column.
+
+    Returns:
+        The formatted column.
+    """
+    if isinstance(col, str):
+        return {"title": col, "id": col.lower(), "type": "str"}
+    elif isinstance(col, (dict,)):
```

Comment:
nit: don't need these `elif` since the `if` cases return. These can just become `if`.